index.html

```diff
@@ -386,6 +386,30 @@
           a representation of the signing date.</dd>
         </dl>
       </section>
+
+      <section>
+        <h2>Revocation Model</h2>
+        <p>A <a>Verifiable Claim</a> may provide revocation information
+        by referring to a revocation list by adding the following
+        <a>property</a> to the <a>Verifiable Claims Model</a>:</p>
+        <dl>
+          <dt><var>revocationList</var></dt>
+          <dd>The format of the revocation list will vary depending on the
+          signature scheme used, but will almost always be a URL to a
+          resource that contains zero or more revocation entries.
```

Comment:
Alright, I'll update based on this feedback and make the entry more generic. I would imagine this feature would end up following what we do for "signature"... that is, you can have different revocation schemes just like you can have different signature schemes. This provides the greatest amount of flexibility. I imagine we'll also define and implement at least one revocation scheme during the VCWG's timeframe.